styleguide/derek/_partials/_footer.jade

```diff
@@ -1,2 +1,2 @@
-.container-fluid.bg-blue-gray.standard-padding
+.container-fluid.bg-blue-gray.fluid-footer-container
```

Comment:
@amberRrucker that's a problem for Drupal, not this styleguide. We should solve Drupal problems in Drupal, and HTML/CSS problems here.